Patch for #510703 from Radovan Garabík (garabik@kassiopeia.juls.savba.sk)

Index: dictd/dictfmt.c
===================================================================
--- dictd.orig/dictfmt.c	2009-01-04 18:19:09.000000000 +0100
+++ dictd/dictfmt.c	2009-01-04 18:19:27.000000000 +0100
@@ -1500,7 +1500,7 @@
 	    }
 
 	    if (indexonly_base64){
-	       i_size = (int) b64_decode (offset);
+	       i_size = (int) b64_decode (size);
 	    }else{
 	       i_size = xatoi (size);
 	    }
